packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -677,6 +678,27 @@ function LineItemSublabelAndPrice( {
 		);
 	}
 
+	if ( isDIFMExtraPageProduct( product ) ) {
+		const itemPrice = product.item_original_cost_for_quantity_one_display;
```

Comment:
I wish we would not have to use snake_case variables in components and elsewhere 🤷

Reply:
Agreed! It does make it a bit confusing. The good part here is that these properties are strongly typed and are part of the `ResponseCartProduct` type.
https://github.com/Automattic/wp-calypso/blob/3b160cb2867073bac5a4306cd53d45912b21a7de/packages/shopping-cart/src/types.ts#L292-L339